docs/guides/dns-over-https.md

```diff
@@ -35,7 +35,18 @@ cloudflared -v
 
 **Note:** Binaries for other operating systems can be found here: <https://developers.cloudflare.com/argo-tunnel/downloads/>
 
-#### ARM architecture (Raspberry Pi)
+#### arm64 architecture (64-bit Raspberry Pi)
+
+Get the download link of `cloudflared-linux-arm64` from <https://github.com/cloudflare/cloudflared/releases/>
+
+```bash
+wget -O cloudflared https://the_link_you_just_get
```

Comment:
This isn't going to work as directions. We will have a number of users open issues because https://the_link_you_just_get doesn't exist.

Reply:
updated, thanks